FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/include/tls_freertos.h

```diff
@@ -75,19 +80,16 @@ typedef struct SSLContext
 
 /**
  * @brief Definition of the network context for the transport interface
- * implementation that uses mbedTLS and FreeRTOS+TCP sockets.
- *
- * @note For this transport implementation, the socket handle and
- * SSL context is used.
+ * implementation that uses mbedTLS and FreeRTOS+TLS sockets.
  */
 struct NetworkContext
 {
-    Socket_t tcpSocket;
+    Socket_t TLSSocket;
```

Comment:
Should this still be the `tcpSocket`?

Reply:
Yes. This will still be tcpSocket. This was also an accidental change and fixed it in a later commit in this branch.